src/main/java/duke/TaskList.java

```diff
@@ -1,3 +1,11 @@
+package duke;
+
+import duke.exceptions.DukeException;
+import duke.task.Deadline;
+import duke.task.Event;
+import duke.task.Task;
+import duke.task.ToDo;
```

Comment:
It is wonderful that you imported the packages one by one and not just import everything wholesale!